openapi/crm/crm.json

```diff
@@ -0,0 +1,223 @@
+                      "fields": [
+                        {
+                          "externalId": "externalId_xxxx",
+                          "type": "FIELD_TYPE_xxxx",
+                          "title": "Last activity date",
+                          "description": "Date and time of last activity"
+                        },
+                        {
+                          "externalId": "externalId_xxxx_xxxx",
+                          "type": "FIELD_TYPE_xxxxx",
+                          "title": "Mobile phone number",
+                          "description": "Mobile phone number of the contact."
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "User Not Found"
+          }
+        },
+        "operationId": "get-field-schema",
+        "description": "Retrieve field schema based on namespace."
+      }
+    },
+    "/{namespace}/{resourceTypeCode}/meta/fields/{fieldId}/options": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "fieldId",
+          "in": "path",
+          "required": true,
+          "description": "Unique identifier for field"
+        }
+      ],
+      "get": {
+        "summary": "Get field options",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
+            "content": {
+              "application/json": {
+                "examples": {
+                  "Get field options": {
+                    "value": {
+                      "options": [
+                        {
+                          "value": "Option 1",
+                          "name": "Option 1"
+                        },
+                        {
+                          "value": "Option 2",
+                          "name": "Option 2"
+                        },
+                        {
+                          "value": "Option 3",
+                          "name": "Option 3"
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          }
+        },
+        "operationId": "get-field-options",
```

Comment:
change operationID to `GetFieldOptions`.

We are going to use `operationId` property to map handler methods for all the `org` related routes in mux.

Reply:
Okay, Changed.